style.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Kumbh+Sans:wght@400;700&display=swap');
+
+body, html {
+    background: url(images/bg-pattern-top.svg), url(images/bg-pattern-bottom.svg), hsl(185, 75%, 39%);
+    background-repeat: no-repeat;
+    background-position: right 52vw bottom 36vh, left 50vw top 52vh;
+    /* overflow: hidden; */
+    font-family: 'Kumbh Sans', sans-serif;
+    height: 100%;
+    margin:0px;
+}
+
+.card {
+    background-color: white;
+    position: fixed;
+    top: 50%;
+    left: 50%;  
+    transform: translate(-50%, -50%);
+    border-radius: 20px;
+    display: flex;
+    flex-direction: column;
+    box-shadow: 0px 0px 20px 10px hsl(185, 75%, 39%);
+ }
+
+ .card .pattern {
+     height: 140px;
+ }
+
+.card .pattern img {
+    height: 100%;
+    width: 100%;
+    border-radius: 20px 20px 0 0;
+}
+
+.card .picture {
+    height: 150px;
+    display: flex;
+    justify-content: center;
+    align-content: space-evenly;
+}
+
+.card .picture img {
+    border-radius: 50%;
+    border: 5px solid white;
+    position: fixed;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.card .picture p {
+    margin-top: 70px;
+    text-align: center;
+    font-size: 15px;
+    line-height: 30px;
+    color: hsl(227, 10%, 46%);
+    font-weight: 400;
+    opacity: 0.8;
+}
+
+.card .picture p span {
+    font-size: 18px;
+}
+
+.card .picture p span span {
+    color: black;
+    font-weight: 700;
+}
+
+.card .data {
+    border-top: 1px solid hsl(0, 0%, 59%);
+    display: flex;
+    flex-direction: row;
+    height: 80px;
+    justify-content: space-evenly;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 20px;
+    color: hsl(227, 10%, 46%);
+    opacity: 0.8;
+    letter-spacing: 1px;
+    text-align: center;
+}
+
+.card .data .stats span {
+    font-size: 18px;
+    color: black;
+    font-weight: 700;
+    letter-spacing: 0px;
+}
+
+@media screen and (max-width: 375px) {
```

Comment:
Stranica se ne prikazuje lepo na telefonima sirine 375px i manje, obrati paznju na sirinu kartice u odnosu na sirinu cele stranice i ispravi to